sql/2019/06_Fonts/06_04.sql

```diff
@@ -0,0 +1,25 @@
+#standardSQL
+
+-- counts font-display value usage
+
+WITH
+  response_bodies AS (
```

Comment:
Rather than inspecting the response bodies, let's rely on the Lighthouse "font-display" audit.

This is used on our website: https://httparchive.org/reports/state-of-the-web#fontDisplay

Here's the underlying timeseries query: https://github.com/HTTPArchive/bigquery/blob/master/sql/timeseries/fontDisplay.sql

So for 2019_07_01 it'd be:

```sql
#standardSQL
SELECT
  ROUND(SUM(IF(JSON_EXTRACT(report, '$.audits.font-display.score') IN ('true', '1'), 1, 0)) * 100 / COUNT(0), 2) AS percent
FROM
  `httparchive.lighthouse.2019_07_01_mobile`
WHERE
  report IS NOT NULL AND
  JSON_EXTRACT(report, '$.audits.font-display.score') IS NOT NULL
```

Reply:
ahhh lighthouse. I see. I will edit mine